db/seeds/books_favorites.js

```diff
@@ -19,8 +19,5 @@ exports.seed = function(knex) {
       .then(() => {
         return knex.raw('ALTER SEQUENCE favorites_id_seq RESTART WITH 1');
```

Comment:
Did you find that you needed to reset the primary key when re-seeding? Usually, this isn't necessary and it's ok to let the primary key keep incrementing. But was something relying on the primary keys to start over?